lib/prompts/apiMock.ts

```diff
@@ -76,9 +90,19 @@ CEO name (string): The name of the CEO
 Response
 ---
 
-Company name: Apply
+${
+  isArray
+    ? `
+Company name: [Apple, Google, Facebook]
+Annual earnings: [1000000, 2000000, 3000000]
+CEO name: [Tim Cook, Sundar Pichai, Mark Zuckerberg]
+`
+    : `
+Company name: Apple
 Annual earnings: 1000000
 CEO name: Tim Cook
+`
+}
 
 -- END OF EXAMPLES --
 
```

Comment:
Please could we write some tests for the line below?

I'm not really sure how it works for objects :confused: 

E.g. if there's a `user` object, would it output `User (object): <description>` as one line? 

```
${Object.entries(responseType)
  .map(([k, v]) => `${k} (${v.type}): ${v.description ?? "no description"}`)
  .join("\n")}
```

Reply:
Objects are already deconstrucuted to a "Properties" type, so this line doesn't need to deal with nested objects, only flat "properties". I've added a test in apiMockPrompt.test.ts to demonstrate this.